src/api/devices.ts

```diff
@@ -14,38 +14,8 @@ const sortDevices = (devices: Device[]) =>
     }
   })
 
-export const SHARED_DEVICE = 'Shared Device'
-
-const createSharedDevice = (dongleId: string): Device => ({
```

Comment:
We can come back and re-add  if we decide we really want to show in the sidebar when viewing a shared device, but we're already about to remove the route listing, so probably not a big deal to match old connect